README.md

```diff
@@ -28,6 +28,30 @@ The key deliverables of the project are as given. The table summarises the relea
 | D3.1.1     | November 15th | Developer documentation |
 | D3.1.2     | November 15th | Test protocol           |
 
+## Instructions to run
+
+The prerequisites for getting the server up and running are as follows:
+
+1. Docker is up and running on your server. You can check it using the command docker ps.
+2. Pre-install [jq](https://jqlang.github.io/jq/), a lightweight and flexible command-line JSON processor for parsing and manipulating JSON data.
+
+You can request a pre-defined configuration file and skip the following steps by contactingg [support@igrant.io](mailto:support@igrant.io). Please specify the desired admin username in your request. Alternatively, you can proceed with steps 3 and 4 for manual installation and configuration.
+
+3. Install keycloak and use the parameters in step 4 configurations below.
+4. The configuration parameters used by the Consent BB API server is created at <server address>/bb-consent-api/resources/config/config-development.json. This sets up the default parameters for the Consent BB server instance, such as organisation details, admin access security (with keycloak, etc. This can also be modified later but would require building the server again.
```

Comment:
Configuration file (.json file) is loaded to the server during startup. It is not available to access from the internet.